src/main/java/org/terasology/launcher/util/FileUtils.java

```diff
@@ -99,7 +99,7 @@ public static boolean extractZipTo(File archive, File outputLocation) {
             try (ZipInputStream zis = new ZipInputStream(new FileInputStream(archive))) {
                 ZipEntry ze;
                 while ((ze = zis.getNextEntry()) != null) {
-                    File extractedFile = new File(outputLocation, com.google.common.io.Files.getNameWithoutExtension(ze.getName()));
+                    File extractedFile = new File(outputLocation, ze.getName());
```

Comment:
@iojw since you approved I guess changing this back is okay/correct? Is what you described int https://github.com/MovingBlocks/TerasologyLauncher/pull/393 actually a non-issue here (the different behavior on different systems)?